webviz_subsurface/plugins/_simulation_time_series/utils/history_vectors.py

```diff
@@ -54,13 +54,15 @@ def create_history_vectors_df(
         if historical_vector_name and historical_vector_name in provider.vector_names():
             historical_vector_and_vector_name_dict[historical_vector_name] = vector
 
-    if not historical_vector_and_vector_name_dict:
+    # Get lowest valid realization number
+    realization = (
+        None if len(provider.realizations()) <= 0 else min(provider.realizations())
+    )
```

Comment:
Is `realizations()` returning a list, or something list-like? If so maybe we can simplfy to this:
```suggestion
    realization = min(provider.realizations(), default=None)
```

Reply:
Absolutely. Way cleaner code!